external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisInputDStream.scala

```diff
@@ -100,6 +103,7 @@ object KinesisInputDStream {
     private var endpointUrl: Option[String] = None
     private var regionName: Option[String] = None
     private var initialPositionInStream: Option[InitialPositionInStream] = None
+    private var initialPositionInStreamTimestamp: Option[Date] = None
```

Comment:
I'm hoping we won't have to take both `initialPositionInStream` and `initialPositionInStreamTimestamp`. The builder is internal APIs, therefore we can definitely change it.